sdk/metric/sdk.go

```diff
@@ -218,7 +218,10 @@ func (s *syncInstrument) acquireHandle(kvs []attribute.KeyValue, labelPtr *attri
 		// needed for the `sortSlice` field, to avoid an
 		// allocation while sorting.
 		rec = &record{}
-		rec.storage = attribute.NewSetWithSortable(kvs, &rec.sortSlice)
+		// Make a copy of `kvs` so that the original set will not be sorted
+		tmpKvs := make([]attribute.KeyValue, len(kvs))
```

Comment:
This is adding an allocation to a highly optimized method. Can we instead take the other option outlined in the link issue and update the function docs to clearly indicate that the labels are mutated?

Reply:
So should I update docs on relevant functions in [sdk.go](https://github.com/open-telemetry/opentelemetry-go/blob/main/sdk/metric/sdk.go) instead?